R/PipeOpTaskSurvClassif.R

```diff
@@ -0,0 +1,71 @@
+PipeOpTaskSurvClassif <- R6Class(
+  "PipeOpTaskSurvClassif",
+  inherit = PipeOpTaskTransformer,
+
+  public = list(
+    initialize = function(id = "trafotask_survclassif") {
+      super$initialize(id = id,
+                       input = data.table::data.table(name = "input", train = "TaskSurv", predict = "TaskSurv"),
+                       output = data.table::data.table(name = c("output", "meta"), train = c("TaskClassif", "*"), predict = c("TaskClassif", "*")))
+    }
+  ),
+
+  private = list(
+    .train = function(task, cut = NULL) {
+      cut = NULL
+      task = task[[1]]
+      data = task
+
+      assert_task(task, task_type = "surv")
```

Comment:
no need => the pipeop constructor checks for the task type that it is indeed `TaskSurv`. Try putting something esle and you will get an error